framework/src/systems/NonlinearSystemBase.C

```diff
@@ -2740,68 +2731,47 @@ NonlinearSystemBase::computeDamping(const NumericVector<Number> & solution,
   Real damping = 1.0;
   bool has_active_dampers = false;
 
-  try
+  if (_element_dampers.hasActiveObjects())
   {
-    if (_element_dampers.hasActiveObjects())
-    {
-      PARALLEL_TRY
-      {
-        TIME_SECTION(_compute_dampers_timer);
-        has_active_dampers = true;
-        *_increment_vec = update;
-        ComputeElemDampingThread cid(_fe_problem);
-        Threads::parallel_reduce(*_mesh.getActiveLocalElementRange(), cid);
-        damping = std::min(cid.damping(), damping);
-      }
-      PARALLEL_CATCH;
-    }
+    TIME_SECTION(_compute_dampers_timer);
+    has_active_dampers = true;
+    *_increment_vec = update;
+    ComputeElemDampingThread cid(_fe_problem);
+    Threads::parallel_reduce(*_mesh.getActiveLocalElementRange(), cid);
+    damping = std::min(cid.damping(), damping);
+  }
 
-    if (_nodal_dampers.hasActiveObjects())
-    {
-      PARALLEL_TRY
-      {
-        TIME_SECTION(_compute_dampers_timer);
+  if (_nodal_dampers.hasActiveObjects())
+  {
+    TIME_SECTION(_compute_dampers_timer);
 
-        has_active_dampers = true;
-        *_increment_vec = update;
-        ComputeNodalDampingThread cndt(_fe_problem);
-        Threads::parallel_reduce(*_mesh.getLocalNodeRange(), cndt);
-        damping = std::min(cndt.damping(), damping);
-      }
-      PARALLEL_CATCH;
-    }
+    has_active_dampers = true;
+    *_increment_vec = update;
+    ComputeNodalDampingThread cndt(_fe_problem);
+    Threads::parallel_reduce(*_mesh.getLocalNodeRange(), cndt);
+    damping = std::min(cndt.damping(), damping);
+  }
+
+  if (_general_dampers.hasActiveObjects())
+  {
+    TIME_SECTION(_compute_dampers_timer);
 
-    if (_general_dampers.hasActiveObjects())
+    has_active_dampers = true;
+    const auto & gdampers = _general_dampers.getActiveObjects();
+    for (const auto & damper : gdampers)
     {
-      PARALLEL_TRY
+      Real gd_damping = damper->computeDamping(solution, update);
+      try
       {
-        TIME_SECTION(_compute_dampers_timer);
-
-        has_active_dampers = true;
-        const auto & gdampers = _general_dampers.getActiveObjects();
-        for (const auto & damper : gdampers)
-        {
-          Real gd_damping = damper->computeDamping(solution, update);
-          try
-          {
-            damper->checkMinDamping(gd_damping);
-          }
-          catch (MooseException & e)
-          {
-            _fe_problem.setException(e.what());
-          }
-          damping = std::min(gd_damping, damping);
-        }
+        damper->checkMinDamping(gd_damping);
+      }
+      catch (MooseException & e)
+      {
+        _fe_problem.setException(e.what());
       }
-      PARALLEL_CATCH;
+      damping = std::min(gd_damping, damping);
     }
   }
-  catch (MooseException & e)
-  {
-    // The buck stops here, we have already handled the exception by
-    // calling stopSolve(), it is now up to PETSc to return a
-    // "diverged" reason during the next solve.
```

Comment:
Why you deleted everything here?

Reply:
Because the git conflict resolution was very confusing!

Reply:
You possibly should keep my changes

Reply:
I think maybe I never merge your stuff, and then I never have conflicts. What do you think?

Reply:
It would be great.